assertions/poster.go

```diff
@@ -153,8 +154,12 @@ func (m *Manager) PostAssertionBasedOnParent(
 			return none, nil
 		}
 		return none, errors.Wrapf(err, "could not get execution state at batch count %d with parent block hash %v", batchCount, parentBlockHash)
+	}
 
+	if err = m.waitToPostIfNeeded(ctx, parentCreationInfo); err != nil {
```

Comment:
We probably need to modify m.ExecutionStateAfterParent to return a boolean if we need to post an overflow asseriton and skip the min assertion period blocks check